recipes/cygwin/meta.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: cygwin
+  version: "2.4.1"    # this is the version of the cygwin dll.  Not sure how relevant it is.
+
+# don't download source here - seems there might be some redirect that leads to a different file.
```

Comment:
I agree with your sentiment.  I did this because the SHA256 for the file downloaded by Conda did not match the one I downloaded and sha256'd myself.  Further, If I change the SHA256 to the one Conda created, then I get an error message that the 16-bit application can't run.  Something is really going wrong.  It may be that conda is trying to extract the exe, but surely it would do that after sha256?